dotnet/samples/KernelSyntaxExamples/Example66_FunctionCallingStepwisePlanner.cs

```diff
@@ -34,7 +34,7 @@ public static async Task RunAsync()
             Console.WriteLine($"Q: {question}\nA: {result.FinalAnswer}");
 
             // You can uncomment the line below to see the planner's process for completing the request.
-            // Console.WriteLine($"Chat history:\n{result.ChatHistory?.AsJson()}");
+            // Console.WriteLine($"Chat history:\n{System.Text.Json.JsonSerializer.Serialize(result.ChatHistory)}");
```

Comment:
It turns out that with the change to tools, printing this isn't that useful anymore, since it doesn't actually show the function name that was chosen. (I think it's in the ChatHistory, but maybe deeper than is included in the default serialization?) I want to find a better solution for that but for now will keep this commented out.